src/utils.ts

```diff
@@ -445,6 +452,9 @@ export function extractChatHistory(memoryVariables: MemoryVariables): [string, s
   return chatHistory;
 }
 
+/**
+ * @deprecated Use noteUtils getLinkedNotes() instead.
+ */
 export function extractNoteTitles(query: string): string[] {
```

Comment:
This is extracting from a query and not a note, so it's not the same as `getLinkedNotes()`

Reply:
Got it. Removed the deprecated annotation.